ui-ngx/src/styles.scss

```diff
@@ -518,6 +518,10 @@ pre.tb-highlight {
     }
   }
 
+  .mat-mdc-icon-button:disabled {
+    opacity: var(--mdc-icon-button-disabled-icon-opacity, 0.38);
+  }
+
```

Comment:
Why are the styles that come with the material unsuitable for this task?
Maybe then it would be better to correct the theme style